container_files/assets/www/doc/builtin/procedures/Accuracy.md

```diff
@@ -83,6 +94,124 @@ with the same row name as in the input, and the following columns:
   * `falseNegatives`, `trueNegatives`, `falsePositives`, `truePositives`
   * `falsePositiveRate`, `truePositiveRate`, `precision`, `recall`
 
+Note that rows with the same score get grouped together.
+
+### <a name="categorical"></a>Categorical mode
+
+The `status` field will contain a sparse confusion matrix along with performance 
+statistics (e.g. [precision, recall](http://en.wikipedia.org/wiki/Precision_and_recall)) 
+for the classifier, where the label with the maximum score will be chosen as the
+prediction for each example. 
+
+The value of `support` is the number of true positives
+for that label. The `weighted_statistics` represents the average of the 
+per-label statistics, weighted by each label's support value.
+
+The `confusion_matrix` is indexed by true label and then by predicted label.
+
+Here is a sample output:
+
+```
+GET http://localhost/v1/procedures/ttnc_test_scorer/runs/1
+
+{
+    "status": {
+        "label_statistics": {
+            "1": {
+                "f1_score": 0.0,
+                "recall": 0.0,
+                "support": 1,
+                "precision": 0.0
+            },
+            "0": {
+                "f1_score": 0.8000000143051146,
+                "recall": 1.0,
+                "support": 2,
+                "precision": 0.6666666865348816
+            },
+            "2": {
+                "f1_score": 1.0,
+                "recall": 1.0,
+                "support": 2,
+                "precision": 1.0
+            }
+        },
+        "weighted_statistics": {
+            "f1_score": 0.7200000057220459,
+            "recall": 0.8,
+            "support": 5,
+            "precision": 0.6666666746139527
+        },
+        "confusion_matrix": {
```

Comment:
hmm we should think about the format here... Perhaps a set of `{actual: "1", predicted: "0", count: 5}` objects in an array?